src/linkAttributes/linkAttributes.ts

```diff
@@ -197,8 +198,8 @@ export function updateVisibleLinks(app: App, plugin: SuperchargedLinks) {
             const cachedFile = app.metadataCache.getFileCache(file);
 
             // @ts-ignore
-            const metadata = leaf.view?.metadataEditor.contentEl;
-            if (!!metadata) {//
+            const metadata = leaf.view?.metadataEditor?.contentEl;
```

Comment:
In versions prior to 1.4.0 (? the properties update, I'm unsure which version specifically) the `metadataEditor` didn't exist. This just makes it safe when it's not present. I wrote this fix when properties was still the insider build so I needed this. Today it's probably not necessary anymore, but it also can't hurt.